internal/config/config.go

```diff
@@ -269,6 +272,7 @@ func parsePeer(p peer) (*Peer, error) {
 			return nil, fmt.Errorf("invalid router ID %q", p.RouterID)
 		}
 	}
+	src := net.ParseIP(p.SrcAddr)
```

Comment:
if this returns nil, I guess the code handles it correctly. Right?

Reply:
Yes, a nil value in `src` will eventually make `localAddressExists()` return `false`.